app/gui/src/presenter/graph/visualization/manager.rs

```diff
@@ -521,6 +419,116 @@ impl Manager {
 }
 
 
+// === synchronize ===
```

Comment:
Is this a section name? If so, it sohuld be written with first capital letter. Also, is this section name undertandable? It might be, based on the context in the file - looking just at this part, I dont understand why implementation for manager i in the "synchronize" section.